src/database/config.js

```diff
@@ -0,0 +1,17 @@
+const dotenv = require('dotenv');
+
+dotenv.config();
+module.exports = {
+  production: {
+    use_env_variable: process.env.DATABASE_URL,
+    dialect: 'postgres',
+  },
+  development: {
+    use_env_variable: process.env.DEV_DATABASE_URL,
+    dialect: 'postgres',
+  },
+  test: {
+    use_env_variable: process.env.DEV_DATABASE_URL,
```

Comment:
@drayzii Can we have a separate test database instead of using the same Database for tests.